Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -1,95 +1,143 @@
-﻿namespace BootCamp.Chapter1
+﻿using System;
+
+namespace BootCamp.Chapter1
 {
     public static class ArrayOperations
     {
-        /// <summary>
-        /// Sort the array in ascending order.
-        /// If array empty or null- don't do anything.
-        /// </summary>
-        /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (IsNullArray(array))
+                return;
+            
+            int temp;
+
+            for (int i = 0; i < array.Length; i++)                  //For each index of the array...
```

Comment:
Neat comments, but formatting should be 1 line above the line that you explain, 
```
// Comment
code
```
Otherwise it's a pain to maintain. Anyways, it serves the purpose- very clear!